.github/workflows/build-package.yml

```diff
@@ -162,10 +172,110 @@ jobs:
+    env:
+      MPYL_CONFIG_PATH: mpyl_config.example.yml
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Install MPyL
+        continue-on-error: true
+        run: pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple mpyl==${{ needs.Build_Package.outputs.mpylVersion }}
+
+      - name: Initialize repo
+        run: mpyl repo init -b ${{ github.ref }} -u https://${{ env.GITHUB_TOKEN }}@github.com/${{ github.repository }}.git -p
+
+      - name: Repo status
+        run: mpyl repo status
+
+      - name: Check mpyl health
+        run: mpyl health --ci
+
+      - name: Lint projects
+        run: mpyl projects lint
+
+      - name: Validate upgrade status
+        run: mpyl projects upgrade
+
+      - name: Build status
+        run: mpyl build status
+
+      - name: Force change
+        run: echo "To force a change " >> tests/projects/service/change.txt
+
+      - name: Build status with change
+        run: mpyl build status
+
+      - name: Build run
+        run: mpyl build run
+
+  Integration_Test_Tag:
+    name: Test tag v${{ needs.Build_Package.outputs.mpylVersion }}
+    runs-on: ubuntu-latest
+    needs: [ Build_Package ]
+    env:
+      MPYL_CONFIG_PATH: mpyl_config.example.yml
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Install MPyL
+        continue-on-error: true
+        run: pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple mpyl==${{ needs.Build_Package.outputs.mpylVersion }}
+
+      - name: Initialize repo
+        run: mpyl repo init -b ${{ github.ref }} -u https://${{ env.GITHUB_TOKEN }}@github.com/${{ github.repository }}.git -p
+
+      - name: Get head revision
+        id: headRevision
+        run: echo "headRevision=$(git rev-parse HEAD)" >> "$GITHUB_OUTPUT"
+
+      - name: Repo status
+        run: mpyl repo status
+
+      - name: Build status
+        run: mpyl build status
+
+      - name: Force change
+        run: |
+          echo "To force a change " >> tests/projects/service/change.txt
+          git add tests/projects/service/change.txt
+
+      - name: Commit the change
+        run: |
+          git config --global user.email "somebody@somewhere.com"
+          git config --global user.name "Some Person"
+          git commit -a -m "A random change"
+
+      - name: Switch to main
+        run: git checkout -b main ${{ steps.headRevision.outputs.headRevision }}
+
+      - name: Merge branch
+        run: git merge --no-ff -m "Merge PR ${{ github.ref }}" ${{ github.ref }}
+
+      - name: Tag merge
+        run: git tag -a v${{ needs.Build_Package.outputs.mpylVersion }} -m "Tagging v${{ needs.Build_Package.outputs.mpylVersion }}"
+
+      - name: Show build status
+        env:
+          TAG_NAME: v${{ needs.Build_Package.outputs.mpylVersion }}
+        run: mpyl build status
```

Comment:
not running the build?

Reply:
We could, but not sure how much the added value is. Besides the change detection mechanism there's no difference with PR builds.

Reply:
https://github.com/Vandebron/mpyl/pull/233/commits/ce747c7c42a25a7c12e44e37a47a3fb09e9b6b51